unittests/AST/ASTBuilder.cpp

```diff
@@ -267,7 +267,25 @@ TEST_SUITE("ASTBuilder::CreateNull") {
   }
 }
 
-TEST_SUITE("ASTBuilder::CreateUndef") {
+TEST_SUITE("ASTBuilder::CreateUndefInteger") {
```

Comment:
Can you merge `TEST_SUITE("ASTBuilder::CreateUndefInteger")` and `TEST_SUITE("ASTBuilder::CreateUndefPointer")` into `TEST_SUITE("ASTBuilder::CreateUndef*")` and have them share some of the common code?